1-exercises/A-accessing-values/exercise3.js

```diff
@@ -21,7 +21,10 @@ let basketballTeam = {
 */
 
 // write code here
-
+topPlayers = basketballTeam.topPlayers.sort();
+for (const player of topPlayers) {
```

Comment:
I like your use of a for loop here Farzad.
I did this: console.log(basketballTeam.topPlayers.join("\n"));
but I prefer your way.